### PR TITLE
KNX sensor value_template

### PR DESCRIPTION
**Pull request by farmio**
## Proposed change

Add value_template configuration option for KNX sensors.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/47734
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -355,7 +355,7 @@ Binary sensors are read-only. To write to the KNX bus configure an exposure [KNX
 ```yaml
 knx:
   binary_sensor:
-    - name: sensor1
+    - name: Sensor 1
```
Comment (frenck) [inline source/_integrations/knx.markdown]: This change is not related to this PR and feature (and a lot of other changes as well). Please do those in a separate PR against the `current` branch. 

Please note; that we have a YAML style guide now, which this change doesn't adhere to either:
<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/knx.markdown (hunk 1 above)
Comment (farmio) [reply source/_integrations/knx.markdown]: regarding the style guide:
> As written in the first chapter, strings are preferably enquoted with double quotes. However, the following value types are exempted from this rule, as is makes our examples more readable:
> - Values that accept a limited set of possible, hardcoded values. For example, mode in automations.

```yaml
sync_state: every 60
```
can be `every | expire` alone or followed by a number (minutes) or `init | true | false`. Or just an integer.
Quoted or not?

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.
